.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
-lib/
+dist/
```

Comment:
Can we keep the name `lib`?

I actually really like it!

Is there a reason we are renaming to `dist`?

Reply:
Hm... well, usually I'd use `lib` for files which were transpiled using `babel` (usually I'd do that for modules meant to run only with node), and `dist` to distribute bundles output by `rollup` (or webpack).
Of course I can still change it if you prefer.